api/apps/file2document_app.py

```
@@ -14,29 +14,88 @@
-from api import settings
-from api.utils.api_utils import get_json_result
+from api.utils.api_utils import (
+    server_error_response,
+    get_data_error_result,
+    validate_request,
+    get_json_result,
+    admin_required,
+)
 
 
 @manager.route('/convert', methods=['POST'])
```

Ruff (0.8.2)

Undefined name `manager`

(F821)